### ISSUE TITLE
[Enhance] A11y: Side nav on Gatsby site is not inside a landmark element

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3254]
**User Story:**
As Spark we want our Gatsby Site Side Navigations to be each be a Landmark so that it can be reached using Landmark Navigation. 

**Notes:**
SME: Bob/Olya

**Current State:**
Axe error: All page content must be contained by landmarks
 
The side nav header ("Installing Spark", etc) is outside the nav element and not inside any other landmark element. This element cannot be reached if you are using Landmark Navigation. Fix is to restructure that side nav so the whole thing is a landmark. Also on Using Spark and Principles pages.

**AC:**
- Gatsby Side Navigation will be a landmark
- Installing Spark Side Nav will be a landmark 
- Using Spark Side Nav will be a landmark 
- Principles Side Nav will be a landmark
- will no longer see axe error.
